src/python/pants/module_mapping.yaml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
I don't think we want to store this as a YAML file, as it makes loading things more complex than necessary. We can simply store as a Python dictionary, like we do here: https://github.com/pantsbuild/pants/blob/824a15daa9f633554b5544ae2c59405601e20e1c/src/python/pants/backend/python/dependency_inference/python_stdlib/py36.py

You can add to `backend/python/dependency_inference` a file like `default_module_mapping.py`, then import that dictionary like you would import any other constant.

Reply:
Done :)